app-frontend/src/app/services/map/map.service.js

```diff
@@ -454,6 +454,12 @@ class MapWrapper {
         if (this.persistedThumbnails.has(scene.id)) {
             return this;
         }
+
+        let options = thumbnailOptions ? Object.assign(
```

Comment:
You don't need the ternary operator here - Object.assign ignores undefined values.

Reply:
you also don't need 3 parameters - 
```
Object.assign({}, {a: 1}, someObject)
```
is equivalent to 
```
Object.assign({a: 1}, someObject);
```
Object.assign works by working left to right over its parameters and overriding values in the first object. We often use Object.assign({}, ...) to create a copy of an object, but there's nothing preventing us from sticking values in the first object when we create it inline.